upgrade/helpers/rhevm.py

```diff
@@ -70,24 +67,33 @@ def create_rhevm_instance(instance_name, template_name, datacenter='Default',
         The password of a rhevm project to login.
     RHEV_URL
         An url to API of rhevm project.
+    RHEV_DATACENTER
+        Datacenter name in RHEV where instance is created
+    RHEV_CLUSTER
+        Cluster name in RHEV where instance is created
 
     :param instance_name: A string. RHEVM Instance name to create.
     :param template_name: A string. RHEVM image name from which instance
         to be created.
+    :param datacenter: A string. Name of the datacenter in rhevm
+    :param cluster: A string. Name of the cluster in rhevm
     :param int timeout: The polling timeout in minutes to create rhevm
     instance.
     """
+    if not datacenter:
+        datacenter = os.environ.get('RHEV_DATACENTER')
+    if not cluster:
+        cluster = os.environ.get('RHEV_CLUSTER')
     rhevm_client = get_rhevm_client()
     template = rhevm_client.templates.get(name=template_name)
     datacenter = rhevm_client.datacenters.get(name=datacenter)
-    quota = datacenter.quotas.get(name=quota)
```

Comment:
so quota is not required now ?

Reply:
No, we had it defined it in old setup , not in new as everyone uses single account

Reply:
cool